lib/clients/client.ex

```diff
@@ -0,0 +1,40 @@
+defmodule GlassFactoryApi.Clients.Client do
+  @moduledoc """
+  Defines the Client of organization.
+  """
+  alias GlassFactoryApi.Clients.Client
+
+  @type t() :: %Client{
+          id: integer(),
+          name: String.t(),
+          archived_at: Date.t(),
+          owner_id: integer(),
+          office_id: integer()
+        }
```

Comment:
I'm thinking of treat identifier fields and date fields as a string instead of integers and dates.

The dates because this way we won't need to parse or handle problems like formatting and time zones, just leave as it is.

Reply:
Sure! I've done the change to string 😄